app/src/pages/admin/projectsPage/projectName/projectName.jsx

```diff
@@ -24,46 +24,79 @@ import { ADMIN_PROJECTS_PAGE_EVENTS } from 'components/main/analytics/events';
 import { SCREEN_XS_MAX_MEDIA } from 'common/constants/screenSizeVariables';
 import { navigateToProjectAction } from 'controllers/administrate/projects';
 import { PROJECT_PAGE } from 'controllers/pages';
-import { assignedProjectsSelector } from 'controllers/user';
+import {
+  assignedProjectsSelector,
+  assignedOrganizationsSelector,
+  setActiveProjectKeyAction,
+  userRolesSelector,
+} from 'controllers/user';
+import { MANAGER, userRolesType } from 'common/constants/projectRoles';
+import { ADMINISTRATOR } from 'common/constants/accountRoles';
 import styles from './projectName.scss';
 
 const cx = classNames.bind(styles);
 
 @connect(
   (state, ownProps) => ({
-    isAssigned: !!assignedProjectsSelector(state)[ownProps.project.projectKey],
+    isProjectAssigned: !!assignedProjectsSelector(state)[ownProps.project.projectKey],
+    isOrganizationAssigned: Object.keys(assignedOrganizationsSelector(state)).some(
+      ({ organizationId }) => organizationId === ownProps.project.organizationId,
+    ),
+    userRoles: userRolesSelector(state),
   }),
   {
     navigateToProject: navigateToProjectAction,
+    setActiveProjectKey: setActiveProjectKeyAction,
   },
 )
 @track()
 export class ProjectName extends Component {
```

Comment:
Should this component be a part of the new page?)
I see the same component used on the Projects page, please update it as well, or use the new component here.
'src/pages/organization/organizationProjectsPage/projectsListTable/projectName'

Reply:
Done.